### PR TITLE
config: Use libexec path rather than hard-coding default shim path.

### PR DESCRIPTION
**Pull request by jodh-intel**
Set the default shim path to be below the prefix set by the Makefile,
rather than just hard-coding it to "/usr/local".

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -41,11 +41,10 @@ const (
 	pauseBinRelativePath        = "bin/pause"
 )
 
-var defaultShimPath = "/usr/local/libexec/cc-shim"
-
 const (
-	qemuLite = "qemu-lite"
-	qemu     = "qemu"
+	qemuLite   = "qemu-lite"
```
Comment (grahamwhaley) [inline config.go]: guess we'll let you get away with the mixed in whitespace change @jodh-intel  (threw my review eyes out for a moment ;-) )

File: config.go [hunk 2]
```diff
@@ -60,6 +59,10 @@ const (
 	hyperstartAgent = "hyperstart"
 )
 
+// XXX: set by build
```
Comment (grahamwhaley) [inline config.go]: 'XXX' - is that intentional @jodh-intel ?

File: config.go [hunk 3]
```diff
@@ -60,6 +59,10 @@ const (
...
+var libExecDir = ""
```
Comment (dlespiau) [inline config.go]: libexecDir maybe

I think it'd be nice to have sane defaults in this variable so we don't have to check for "" everywhere. NUL value not allowed! :)

**Comment by grahamwhaley** [comment]
minor comments. Technically looks fine to me afaict
lgtm

**Comment by sameo** [comment]
LGTM

**Pull request merged**